packages/core/src/abstract-dialect/query-interface.js

```
@@ -73,6 +73,7 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
    */
   // TODO: remove "schema" option from the option bag, it must be passed as part of "tableName" instead
   async createTable(tableName, attributes, options, model) {
+    console.log('=== CREATE TABLE TRIGGERED ===', tableName);
```

ESLint

[error] Unexpected console statement.

(no-console)